ios/Podfile

```
@@ -1,20 +1,39 @@
+end
+
+pod 'lottie-ios', '~> 3.4.0'
+
 target 'Cypherd' do
   config = use_native_modules!
   $RNFirebaseAnalyticsWithoutAdIdSupport = true
+  use_frameworks! :linkage => :static
+  $RNFirebaseAsStaticFramework = true
+  
+  # Move permissions setup before use_react_native
+  permissions_path = '../node_modules/react-native-permissions/ios'
```

rubocop (1.67.0)

[warning] Useless assignment to variable - `permissions_path`. Did you mean `permission_pods`?

(Lint/UselessAssignment)